project1/styles.css

```
@@ -0,0 +1,174 @@
+/* FONTS
+	- default body text  mrs-eaves, "Palatino Linotype", "Book Antiqua", Palatino, serif;
+    - date & cite        mr-eaves-modern, "Futura", Helvetica, Arial, sans-serif;
+*/
+
+@import url("https://use.typekit.net/ckw4tit.css");
+
+/* HIGH-LEVEL SETTINGS */
+
+body {
+    font-family: mrs-eaves, "Palatino Linotype", "Book Antiqua", Palatino, serif;
+    font-weight: 400;
+    margin: 0 auto;
+    max-width: 1000px;
+    padding: 0 100px;
+}
+
+h1 {
+    display: flex;
+    font-size: 78px;
+    justify-content: center;
+    margin: 0;
+    padding: 50px 0 0 0;
```

Consider avoiding the `padding` shorthand when you're only looking to set one value:
```suggestion
    padding-top: 50px;
```
This makes it clearer what you're trying to do and, in the long run, gives you cleaner, less complicated CSS.